include/rocksdb/file_system.h

```diff
@@ -160,6 +160,8 @@ struct FileOptions : EnvOptions {
   // handoff during file writes.
   ChecksumType handoff_checksum_type;
 
+  bool report_iostat = true;
+
```

Comment:
I might have missed it, but do not see where this is used.

If it is unsed/necessary and not an accident, it needs a comment (and documented in HISTORY since it is public.